src/AgGrid.elm

```diff
@@ -160,11 +160,22 @@ type Alignment
 
 {-| Possible options for the status bar panels.
 -}
-type StatusPanel
-    = TotalRowCount
-    | TotalAndFilteredRowCount
-    | FilteredRowCount
-    | SelectedRowCount
+type StatusBarPanel
+    = TotalRowCount Alignment
+    | TotalAndFilteredRowCount Alignment
+    | FilteredRowCount Alignment
+    | SelectedRowCount Alignment
+    | Aggregation Alignment (List StatusPanelAggregation)
```

Comment:
Why is the `Alignment` now on the type? If all types have this attribute, we can keep it separate. For the Aggregations this is fine